main.py

```diff
@@ -1,9 +1,185 @@
-print("Wyświetlenie listy zrobionych zadań: 1")
-print("Wyświetlenie listy zadań do wykonania: 2")
-print("Dodaj zadanie: 3")
-while True:
-    odpowiedz = input('Wybierz menu ')
-    if odpowiedz == "3":
-        nowe_zadanie = input("Napisz jakie zadanie chcesz dodać: ")
-        print(nowe_zadanie)
-        break
\ No newline at end of file
+import sqlite3
+from sqlite3 import Error
+from task import task
+
+task_list = []
+database = r"C:\Users\mateusz.saganowski\PycharmProjects\pythonProject\pythonsqlite.db"
+
+
+def create_connection(db_file):
+    """ create a database connection to the SQLite database
+        specified by db_file
+    :param db_file: database file
+    :return: Connection object or None
+    """
+    conn = None
+    try:
+        conn = sqlite3.connect(db_file)
+        return conn
+    except Error as e:
+        print(e)
+
+    return conn
+
+
+def create_table(conn, create_table_sql):
+    """ create a table from the create_table_sql statement
+    :param conn: Connection object
+    :param create_table_sql: a CREATE TABLE statement
+    :return:
+    """
+    try:
+        c = conn.cursor()
+        c.execute(create_table_sql)
+    except Error as e:
+        print(e)
+
+
+def create_task(conn, task):
```

Comment:
Tutaj używasz nazwy zmiennej task, a masz też klasę task, którą importujesz z task.py. Zwróć uwagę, że to `task` wprwoadzasz do tabeli jako `name`. Czy nie lepiej byłoby zmienić nazwę argumentu z task na task_name? task jest mylne, bo brzmi jakbyś całą klase przekazywał, a tego nie robisz.

Reply:
Zgadza się, kod bardziej przejrzysty i czytelny. poprawiono